src/lib.rs

```diff
@@ -246,6 +246,8 @@ pub mod codecs {
     pub mod tiff;
     #[cfg(feature = "webp")]
     pub mod webp;
+    #[cfg(feature = "qoi")]
+    pub mod qoi;
```

Comment:
rustfmt wants these kept in alphabetical order. (That's why our CI is reporting "Some checks were not successful")